Libraries/Lists/VirtualizedList.js

```diff
@@ -1670,6 +1735,20 @@ class CellRenderer extends React.Component<
     this.props.onUnmount(this.props.cellKey);
   }
 
+  _onLayout = (e): void =>
+    this.props.onLayout &&
+    this.props.onLayout(e, this.props.cellKey, this.props.index);
+
+  shouldComponentUpdate(nextProps, nextState) {
+    if (!shallowEqual(this.props, nextProps, ['parentProps'])) {
```

Comment:
Can you use `deepEquals` with a depth limit of 2 instead? That will do a shallow compare of all the props, including `parentProps`. Hopefully that works and you can get rid of your `shallowEqual` fork. If that doesn't work for some reason, could you at least rename to `shallowEqualWithExceptions` and put it in a util folder?

Reply:
@sahrens just to be clear, you are referring to deepDiffer `Libraries/Utilities/differ/deepDiffer` ?